Tests/Command/FixMediaContextCommandTest.php

```diff
@@ -47,20 +45,29 @@ class FixMediaContextCommandTest extends CommandTest
     private $pool;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject|ContextManagerInterface
+     * @var \PHPUnit_Framework_MockObject_MockObject|\Sonata\ClassificationBundle\Model\ContextManagerInterface
```

Comment:
Doesn't adding a use statement like that defeat the purpose of this pull request (i.e., decouple MediaBundle and ClassificationBundle)?

Or am I missing something? My apologies if I am. Looking at this on my phone and it isn't the most convenient way to look at the code.